src/components/InventoryTabs/ConventionalSystems/ConventionalSystemsTab.js

```diff
@@ -156,6 +158,15 @@ const ConventionalSystemsTab = ({
     setAddHostGroupModalOpen
   );
 
+  const REQUIRED_PERMISSIONS = [GENERAL_GROUPS_READ_PERMISSION];
+
+  const { hasAccess: hasGroupsAccess } = usePermissionsWithContext(
+    REQUIRED_PERMISSIONS,
+    true
+  );
```

Comment:
I think the call to usePermissionsWithContext would also need the third argument which is responsible for checking resource definitions. If we provide the third argument as `false`, then your function would check for any `inventory:groups:read` permissions including those that resource definitions (= narrowed to some specific groups only):

https://github.com/RedHatInsights/frontend-components/blob/master/packages/utils/doc/RBAC.md#resource-definitions

Reply:
thanks, thats perfect